scenarios/kubernetes_e2e.py

```diff
@@ -152,17 +175,19 @@ def main(args):
     PARSER = argparse.ArgumentParser()
     PARSER.add_argument(
         '--env-file', action="append", help='Job specific environment file')
+    PARSER.add_argument(
+        '--env-template', help='Job specific environment template')
```

Comment:
I am not super convinced I like the complexity of --env-template. It makes it harder to understand how this scenario works. 

Could you solve this by making